src/main/resources/application.conf

```diff
@@ -299,6 +299,11 @@ openolitor {
       aws-secret-acccess-key = "thedefaultminiosecretneedstobechanged"
     }
 
+    mail {
+      invite-expiration-time-in-days = 90
```

Comment:
just a suggestion for future changes: we could also use durations like `90 days` and load them as such in the code

Reply:
didn't know. Thanks!